scripts/benchmark_migration.py

```diff
@@ -172,6 +172,7 @@ def main(
         rows = session.query(model).count()
         print(f"- {model.__name__} ({rows} rows in table {model.__tablename__})")
         model_rows[model] = rows
+    session.close()
```

Comment:
👍 this makes sense to me. Reset session here so that we can continue to use it below.